lib/PrefsFeed.jsm

```diff
@@ -103,6 +103,9 @@ this.PrefsFeed = class PrefsFeed {
     values.isPrivateBrowsingEnabled = PrivateBrowsingUtils.enabled;
     values.platform = AppConstants.platform;
 
+    // Get the firefox accounts url to send firstrun metrics to.
+    values.fxa_endpoint = Services.prefs.getStringPref("identity.fxaccounts.remote.root", "https://accounts.firefox.com");
```

Comment:
Sorry, I missed this :(

browser.newtabpage.activity-stream.fxaccounts.endpoint is the updated pref